lib/alipay/service.rb

```diff
@@ -1,6 +1,6 @@
 module Alipay
   module Service
-    GATEWAY_URL = 'https://mapi.alipay.com/gateway.do'
+    GATEWAY_URL = 'https://api-sea-global.alipayplus.com/gateway.do'
```

Comment:
perhaps we can try to declare an `attr_accessor` in the `alipay.rb` instead of hardcoding the URL, so we can accessed globally with something like this `Alipay.legacy_gateway_url`, it's easier to configure later with different value without the need to change in the gem code (eg: configure in rails initializer)

https://github.com/kfit-dev/alipay/blob/63959f812c65090a811dd9aeaec1b5a122d66e96/lib/alipay.rb#L25